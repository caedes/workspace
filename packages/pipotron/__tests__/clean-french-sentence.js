import cleanFrenchSentence from '../src/clean-french-sentence';

describe('cleanFrenchSentence', () => {
  it('writes comma correctly', () => {
    expect(cleanFrenchSentence(' ,')).toMatchSnapshot();
  });

  it('writes dot correctly', () => {
    expect(cleanFrenchSentence(' .')).toMatchSnapshot();
  });

  'aeiouyÃ©'.split('').forEach(vowel => {
    it(`replaces "# ${vowel}"`, () => {
      expect(cleanFrenchSentence(`# ${vowel}`)).toMatchSnapshot();
    });
  });

  'bcdfghklmnpqrstvwxz'.split('').forEach(consonant => {
    it(`replaces "# ${consonant}"`, () => {
      expect(cleanFrenchSentence(`# ${consonant}`)).toMatchSnapshot();
    });
  });
});
