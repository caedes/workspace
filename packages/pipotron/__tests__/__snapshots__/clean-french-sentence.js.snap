// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cleanFrenchSentence replaces "# a" 1`] = `"'a"`;

exports[`cleanFrenchSentence replaces "# b" 1`] = `"e b"`;

exports[`cleanFrenchSentence replaces "# c" 1`] = `"e c"`;

exports[`cleanFrenchSentence replaces "# d" 1`] = `"e d"`;

exports[`cleanFrenchSentence replaces "# e" 1`] = `"'e"`;

exports[`cleanFrenchSentence replaces "# f" 1`] = `"e f"`;

exports[`cleanFrenchSentence replaces "# g" 1`] = `"e g"`;

exports[`cleanFrenchSentence replaces "# h" 1`] = `"e h"`;

exports[`cleanFrenchSentence replaces "# i" 1`] = `"'i"`;

exports[`cleanFrenchSentence replaces "# k" 1`] = `"e k"`;

exports[`cleanFrenchSentence replaces "# l" 1`] = `"e l"`;

exports[`cleanFrenchSentence replaces "# m" 1`] = `"e m"`;

exports[`cleanFrenchSentence replaces "# n" 1`] = `"e n"`;

exports[`cleanFrenchSentence replaces "# o" 1`] = `"'o"`;

exports[`cleanFrenchSentence replaces "# p" 1`] = `"e p"`;

exports[`cleanFrenchSentence replaces "# q" 1`] = `"e q"`;

exports[`cleanFrenchSentence replaces "# r" 1`] = `"e r"`;

exports[`cleanFrenchSentence replaces "# s" 1`] = `"e s"`;

exports[`cleanFrenchSentence replaces "# t" 1`] = `"e t"`;

exports[`cleanFrenchSentence replaces "# u" 1`] = `"'u"`;

exports[`cleanFrenchSentence replaces "# v" 1`] = `"e v"`;

exports[`cleanFrenchSentence replaces "# w" 1`] = `"e w"`;

exports[`cleanFrenchSentence replaces "# x" 1`] = `"e x"`;

exports[`cleanFrenchSentence replaces "# y" 1`] = `"'y"`;

exports[`cleanFrenchSentence replaces "# z" 1`] = `"e z"`;

exports[`cleanFrenchSentence replaces "# é" 1`] = `"'é"`;

exports[`cleanFrenchSentence writes comma correctly 1`] = `","`;

exports[`cleanFrenchSentence writes dot correctly 1`] = `"."`;
